istioctl/cmd/analyze.go

```diff
@@ -225,34 +227,27 @@ func Analyze() *cobra.Command {
 				return err
 			}
 
-			// Maybe output details about which analyzers ran
-			if verbose {
-				fmt.Fprintf(cmd.ErrOrStderr(), "Analyzed resources in %s\n", analyzeTargetAsString())
-
-				if len(result.SkippedAnalyzers) > 0 {
-					fmt.Fprintln(cmd.ErrOrStderr(), "Skipped analyzers:")
-					for _, a := range result.SkippedAnalyzers {
-						fmt.Fprintln(cmd.ErrOrStderr(), "\t", a)
+			outputMessages := make([]diag.Message, 0)
+			if len(result.MultiClusterResults) > 0 {
```

Comment:
Will this len be 0 in the single cluster case?  It seemed like that might be the case from line 154 of istiod_analyze.go.   I didn't fully analyze since it's a Draft.

Reply:
Yes, it will maintain the same behavior as before if it's doing single cluster analysis. Only specifying multicluster contexts flag will lead to `MultiClusterResults` being greater than 0.